pkg/config/agent/config.go

```diff
@@ -102,14 +102,21 @@ type AgentConfig struct {
 	TrafficEncryptionMode string `yaml:"trafficEncryptionMode,omitempty"`
 	// WireGuard related configurations.
 	WireGuard WireGuardConfig `yaml:"wireGuard"`
+	// Eanble bridging mode of Pod network on Nodes, in which the Node's transport interface is connected
```

Comment:
```suggestion
	// Enable bridging mode of Pod network on Nodes, in which the Node's transport interface is connected
```

Reply:
Thought I pushed the fix, but seems not..